src/test/java/core/basesyntax/CalculatorTest.java

```diff
@@ -0,0 +1,92 @@
+package core.basesyntax;
+
+import core.basesyntax.exeption.DataProcessingException;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class CalculatorTest {
+
+    private static final int DELTA = 2;
```

Comment:
```suggestion
    private static final double DELTA = 0.01;
```
дельта передається в значенні double і це не кількість знаків заокруглення а похибка (спробуй свої дані з різною дельтою)
delta - the maximum delta between expected and actual for which both numbers are still considered equal.

Reply:
ок, не знала этого